benchmarking/smartseq2/confounding_factor_analysis/Confounding_factor_analysis.R

```diff
@@ -0,0 +1,325 @@
+# R script to do metrics test
+# loading library
+library(ggplot2)
+library(MASS)
+library(plyr)
+library(reshape2)
+library(ggpubr)
+library(gridExtra)
+library(ggpmisc)
+library(cowplot)
+library(corrplot)
+library(ggrepel)
+library(optparse)
+library(rsvd)
+set.seed(42)
+
+# if p<0.05 or -1*log(0.05,base=10) >1.3, return 1 value.
+convert2SigLevels <- function(P) {
```

Comment:
Actually, are you using this function anywhere?

Reply:
Yea, I probably don't need it anymore. just removed it. Thanks